src/main/java/net/minecraftforge/fml/common/registry/VillagerRegistry.java

```diff
@@ -160,7 +162,6 @@ public void registerVillageCreationHandler(IVillageCreationHandler handler)
      * @param defaultSkin
      */
     @SideOnly(Side.CLIENT)
-    @Deprecated // Doesn't work at all.
```

Comment:
It is still marked as deprecated, I just removed the comment about it doing nothing, as it is actually used by the RenderVillager